Update rails

do-you-dare opened the pull request:
This updates rails to Rails 5. The ci checks will fail for rubies 2.0 and 2.1.5, since rails doesn't support them anymore.

Commit:
Add gistadigo gem

This gem eases the job of upgrading the application gems.

Commit:
Auto updated: byebug

Commit:
Auto updated: capistrano

Commit:
Auto updated: capistrano-rails

Commit:
Auto updated: codeclimate-test-reporter

Commit:
Auto updated: daemons

Commit:
Auto updated: database_cleaner

Commit:
Auto updated: jbuilder

Commit:
Auto updated: pg

Commit:
Auto updated: psych

Commit:
Auto updated: spring

Commit:
Auto updated: activejob

Commit:
Auto updated: churn

Commit:
Auto updated: code_analyzer

Commit:
Auto updated: codeclimate

Commit:
Auto updated: libv8

Commit:
Auto updated: mail

Commit:
Auto updated: rails_best_practices

Commit:
Auto updated: rdoc

Commit:
Auto updated: sprockets

Commit:
Require gisdatigo ~>0.1.0

Commit:
Update rspec-rails to 3.5.1

Commit:
Update delayed_job_active_record

Commit:
Update codeclimate gem

It required activesupport ~>4.2, while rails 5 requires 5.0.0.1.

Commit:
Update rails gem

Commit:
Update config files for rails 5

The changes on Rails break some of the code, and the tests with them,
that's why they are pending.

Commit:
Update Active Record Models inheritance

As of rails 5, Active Record Models inherit from ApplicationRecord by
default.

Commit:
Restore routes configuration

The rails 5 update task overwrites the routes configuration.

Commit:
Update jobs inheritance

As of Rails 5, jobs inherit from ApplicationJob, not ActiveJob::Base.

Commit:
Restore exception_notification configurations

Commit:
Restore sample file copying on setup

rafamanzo commented:
What do you think about removing Travis' build for those older breaking version?

rafamanzo commented:
What is the status on WIP acceptance test?

Commit:
Disable ci checks for older rubies

Activesupport doesn't support rubies 2.2 and 2.1.5, and it makes no
sense to test against something we know that won't and shouldn't pass.

dread-uo commented:
Sorry, for a moment I forgot I wip-ed them. So, only one of them is passing. The `root_module_result` of processings is always nil.
